src/client/src/orangehrmDashboardPlugin/components/MyActionSummaryWidget.vue

```diff
@@ -194,7 +194,7 @@ export default {
       navigate('/performance/searchEvaluatePerformanceReview');
     },
     onClickPendingInterview() {
-      navigate('/recruitment/viewCandidates');
+      navigate('/recruitment/viewCandidates?statusId=4');
```

Comment:
![image](https://user-images.githubusercontent.com/12880059/193540971-8bc01031-2886-4023-9ef9-74c3a3999a68.png)
@ManulMax I think we specifically discussed not to hardcode like this